src/test/java/apicurito/tests/steps/VerificationSteps/AssertAll.java

```diff
@@ -0,0 +1,19 @@
+package apicurito.tests.steps.VerificationSteps;
+
+import org.assertj.core.api.SoftAssertions;
+
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+public class AssertAll extends CollectorHelper {
```

Comment:
You extend `CollectorHelper` to access a static variable? Why? I don't think that is right.

Reply:
Because otherwise, I cannot access the variable.

Reply:
It would probably be much better to implement the collector using the singleton pattern.

However, I agree that using a collector everywhere is not great. There are certainly cases where it is useful, but it should not be the default - in general, tests should fail fast, since most of the times, continuing with the tests when one of the assertions fails is meaningless.